### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by regexowl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,27 +81,61 @@ The UI should be running on
...
+```bash
+npm run lint
+```
+
+Any errors that can be fixed automatically, can be corrected by running:
+```bash
+npm run lint --fix
+```
+
+All the linting rules and configuration of eslint can be found in `.eslintrc.yml`
```
Comment (ochosi) [inline README.md:120]: Could add a file link here

File: README.md [hunk 2]
```diff
@@ -71,27 +81,61 @@ The UI should be running on
...
+### Additional eslint rules
+There are also additional rules added to enforce code style. Those being:
+- `import/order` -> enforces the order in import statements and separates them into groups based on their type
+- `prefer-const` -> enforces use of `const` declaration for variables that are never reassigned
+- `no-console` -> throws an error for any calls of `console` methods leftover after debugging
 
 ## Test Guidelines
 
-Testing is done using [React Testing Library](https://testing-library.com/docs/react-testing-library/intro). 
+This project is tested using [Jest](https://jestjs.io/docs/getting-started) framework and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) and [Mock Service Worker](https://mswjs.io/docs/) libraries.
```
Comment (ochosi) [inline README.md:130]:
```suggestion
This project is tested using the [Jest](https://jestjs.io/docs/getting-started) framework, [React Testing Library](https://testing-library.com/docs/react-testing-library/intro), and the [Mock Service Worker](https://mswjs.io/docs/) library.
```

File: README.md [hunk 3]
```diff
@@ -71,27 +81,61 @@ The UI should be running on
...
+
 All UI contributions must also include a new test or update an existing test in order to maintain code coverage.
 
-Tests can be run with
-```
-    npm run test
+### Running the tests
+
+To run the unit tests, and lint and code coverage check run:
```
Comment (ochosi) [inline README.md:136]:
```suggestion
To run the unit tests, the linter, and the code coverage check run:
```

File: README.md [hunk 4]
```diff
@@ -71,27 +81,61 @@ The UI should be running on
...
+| Directory | Description |
+| --------- | ----------- |
+| `/config` | webpack configuration |
+| `/devel` | tools for local development |
+| `/src` | source code |
+| `/src/Components` | source code split by individual components |
+| `/src/test` | test utilities |
+| `/src/test/mocks` | mock handlers and server config for MSW |
+| `/src/store` | Redux store |
+| `/src/api.js` | API calls |
```
Comment (ochosi) [inline README.md:102]: Maybe add a file link for convenience?
Not sure if directories can be linked to as well, but if so I think that'd be great for user-friendliness.

**Review by ochosi — CHANGES_REQUESTED** [review]
Thanks - this looks great and covers a lot of ground on what we discussed during the hackathon!

The nitpicks I mention are mostly about usability or grammar, not the actual content.

File: README.md (hunk 1 above)
Comment (regexowl) [reply README.md]: Added :+1:

File: README.md (hunk 2 above)
Comment (regexowl) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (regexowl) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (regexowl) [reply README.md]: Added the links

**Comment by regexowl** [comment]
Thanks for the comments, @ochosi ! I've added the links and updated the wording.

**Review by ochosi — APPROVED** [review]
Great! LGTM

**Pull request merged**